.github/workflows/ci.yml

```
@@ -35,6 +35,12 @@ jobs:
         # Clone the repository in $GITHUB_WORKSPACE
         - uses: actions/checkout@master  
 
+        - uses: conda-incubator/setup-miniconda@v2
+          with:
+            mamba-version: "*"
+            channels: conda-forge,defaults
+            channel-priority: true
```

```suggestion
        - uses: conda-incubator/setup-miniconda@v2
          with:
            mamba-version: "*"
            channels: conda-forge,defaults
            channel-priority: true
            environment: wearables
            python-version: 3.8
```

Pinning the python version to avoid surprises